src/lib/Icons/Icons.docs.mdx

```diff
@@ -4,45 +4,40 @@ import * as CloseStory from "./IconClose.stories.svelte";
 import * as SearchStory from "./IconSearch.stories.svelte";
 import { Meta, Title, Canvas, Story, Controls, IconGallery, IconItem } from "@storybook/blocks";
 
-<Meta title="Icons/Icons" of={PlusStories} />
+<Meta title="Icons" />
```

Comment:
ahhh I see what's happening here -- still figuring out how mdx files talk to stories fully. setting this as a top-level docs file works so well, appreciated!